homeassistant/components/haveibeenpwned/sensor.py

```diff
@@ -142,13 +146,11 @@ def update_no_throttle(self):
     def update(self, **kwargs):
         """Get the latest data for current email from REST service."""
         try:
-            url = "{}{}".format(URL, self._email)
-
+            url = "{}{}?truncateResponse=false".format(URL, self._email)
+            header = {USER_AGENT: HA_USER_AGENT, "hibp-api-key": self._api_key}
             _LOGGER.debug("Checking for breaches for email: %s", self._email)
-
-            req = requests.get(
-                url, headers={USER_AGENT: HA_USER_AGENT}, allow_redirects=True,
-                timeout=5)
+            req = requests.get(url, headers=header,
```

Comment:
It would be good to switch to a pypi package. I haven't found one that supports api version 3.

Reply:
While I agree a pypi package is probably a good idea in most use cases, in this case it's a single API call. Installing and loading an entire library for a single API call feels a bit heavy handed to me personally.